demo/midgard-sdk/src/tx-builder/index.ts

```diff
@@ -10,3 +10,4 @@ export * as UserEventProtocol from "./user-event-protocol/index.js";
 export * as HubOracle from "./hub-oracle/index.js";
 export * as LinkedList from "./linked-list.js";
 export * as LedgerState from "./ledger-state.js";
+export * as Deposit from "./deposit/types.js";
```

Comment:
We should adhere to the current structure for now. So arrangement of the files should be such that this upholds:
```suggestion
export * as Deposit from "./user-events/deposit/index.js";
```